create.html

```diff
@@ -5,10 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/e98346ef52.js"
+      crossorigin="anonymous"
+    ></script>
     <title>Quiz App - Create</title>
   </head>
   <body>
-    <header><h1>Create</h1></header>
+    <header>
+      <h1>Create</h1>
+    </header>
     <main>
       <!-- Formular Ausgabe funktioniert noch nicht, Java Script in index.html fehlt -->
```

Comment:
This is a nice hint for other developer, but not really necessary as one can see that this is only the HTML structure with CSS for now. I recommend to remove the attributes that you don't need like `action` and ` method`.